lib/whitehall_importer.rb

```diff
@@ -6,8 +6,7 @@ module WhitehallImporter
   def self.create_migration(organisation_content_id, document_type)
     whitehall_migration = ActiveRecord::Base.transaction do
       record = WhitehallMigration.create!(organisation_content_id: organisation_content_id,
-                                 document_type: document_type,
-                                 start_time: Time.current)
+                                 document_type: document_type)
```

Comment:
I appreciate that you haven't formatted these lines but I find its a little jarring to read, could the indentation be changed to something like: ?
```
WhitehallMigration.create!(organisation_content_id: organisation_content_id,
                           document_type: document_type)
```

Reply:
yeah sure thing. I've adjusted it